OpenRA.Mods.Common/Traits/World/ActorMap.cs

```diff
@@ -140,7 +140,7 @@ public void Tick(ActorMap am)
 				currentActors.Clear();
 				currentActors.UnionWith(
 					am.ActorsInBox(position - delta, position + delta)
-					.Where(a => (a.CenterPosition - position).HorizontalLengthSquared < range.LengthSquared
+					.Where(a => (a.CenterPosition - position).HorizontalLengthSquared <= range.LengthSquared
```

Comment:
Leaving this comment for other reviewers: This will need some consideration as well. (As it impacts more than just docking.)